src/pages/index.js

```diff
@@ -66,20 +108,85 @@ function renderCard(data) {
 /*-------------------------------------------------------------------------------*/
 
 function handleProfileFormSubmit(data) {
+  profileEditPopup.setLoading(true);
   userInfo.setUserInfo(data);
+  api.updateUserInfo(userInfo.getUserInfo())
+  .catch((err) => {
+    console.error(err);
+  })
+  .finally(() => profileEditPopup.setLoading(false));
   profileFormValidator.resetValidation();
   profileEditPopup.close();
 }
 
+function handleAvatarFormSubmit(data) {
+avatarEditPopup.setLoading(true);
+const avatar = data.link;
+userInfo.setAvatar({ avatar });
+api.updateAvatar(data.link)
+.catch((err) => {
+  console.error(err);
+})
+.finally(() => avatarEditPopup.setLoading(false));
+avatarFormValidator.resetValidation();
```

Comment:
`resetValidation` is for clearing possible validation errors when you open the **profile** form, because you insert **valid** data into its inputs. But when you add a new card or change the `avatar`, you don't have any errors after submitting. You need to call `disableButton` there.